Python/Github_Issues_PRs_Scrapper/master_fetch_script.py

```diff
@@ -0,0 +1,92 @@
+from github import Github
+import json
+import sys
+
+
+def RetriveIssueData(storeHere, repoName):
+    ''' The funtion retrives the data of issues and returns an array '''
+    issues = repoName.get_issues(state='all')
+
+    for i in issues:
+        issue = {}
+        issue['Title'] = i.title
+        issue['Creator'] = i.user.login
+        issue['Assignees'] = [name.login for name in i.assignees]
+        issue['OpenedAt'] = i.created_at.strftime("%d-%m-%G")
+        if i.closed_at is not None:
+            issue['ClosedAt'] = i.closed_at.strftime("%d-%m-%G")
+        else:
+            issue['ClosedAt'] = None
+        if i.pull_request is not None:
+            issue['PullRequest'] = i.pull_request.html_url
+        else:
+            issue['PullRequest'] = None
+        issue['Body'] = i.body
+        issue['Comments'] = [[j.id, j.user.login] for j in i.get_comments()]
+        storeHere.append(issue)
+
+    return storeHere
+
+
+def RetrivePullRequestData(storeHere, repoName):
+    ''' The funtion retrives the data of PRs and returns an array '''
+    PRs = repoName.get_pulls(state='all')
+
+    for i in PRs:
+        pr = {}
+        pr['ID'] = i.id
+        pr['Creator'] = i.user.login
+        pr['State'] = i.state
+        pr['OpenedAt'] = i.created_at.strftime("%d-%m-%G")
+        if i.merged_at is not None:
+            pr['MergedAt'] = i.merged_at.strftime("%d-%m-%G")
+        else:
+            pr['MergedAt'] = None
+        if i.closed_at is not None:
+            pr['ClosedAt'] = i.closed_at.strftime("%d-%m-%G")
+        else:
+            pr['ClosedAt'] = None
+        rews = []
+        for rew in i.get_review_requests():
+            for k in rew:
+                rews.append([k.id, k.login])
+        pr['Body'] = i.body
+        pr['Reviewers'] = rews
+        pr['FilesChanged'] = i.changed_files
+        if i.merged_by is not None:
+            pr['MergedBy'] = i.merged_by.login
+        else:
+            pr['MergedBy'] = None
+        storeHere.append(pr)
+
+    return storeHere
+
+
+def WriteToJSON(data, fileName):
+    ''' This function converts the array of dicts to json and stores it to .json file '''
+    with open(str(fileName) + ".json", "w") as file:
+        json.dump(data, file, indent=6)
+        file.close()
+    print("Successfully created " + str(fileName) + ".json")
+
+
+if __name__ == "__main__":
+
+    # Setup your github token before running the script
+    g = Github("***********IGHRh*****************hQ")
```

Comment:
Can you create a `.env` file and store the token there? Use a module like `decouple` to import them in your script and use them on the fly.

Also don't push `.env` on the PR. Instead push a `.env.example` highlighting the token that the user needs to keep.